Source/WebSocket/Websocketpp/x509_cert_utilities.hpp

```diff
@@ -94,8 +98,49 @@ bool verify_cert_chain_platform_specific(asio::ssl::verify_context &verifyCtx, c
         certChain.push_back(std::move(certData));
     }
 
-    auto verify_result = verify_X509_cert_chain(certChain, hostName);
+    auto verify_result = false;
+
+#if HC_PLATFORM == HC_PLATFORM_LINUX
+    X509_STORE *store = X509_STORE_new();
+    X509_STORE_CTX_trusted_stack(storeContext, certStack);
+    SSL_CTX *sslContext = SSL_CTX_new(TLS_method());
+    store = SSL_CTX_get_cert_store(sslContext);
+
+    if (sslContext == NULL) {
+        return verify_result;
+    }
+
+    int ret = X509_STORE_set_default_paths(store);
+    if (ret != 1)
+    {
+        return verify_result;
+    }
+
+    ret = X509_STORE_load_locations(store, NULL, "/etc/ssl/certs");
+    if (ret != 1)
+    {
+        printf("\n FAILED TO LOAD CERTS \n");
```

Comment:
no printf
if you need to log something use the tracing macros

Reply:
this was for debugging. I was supposed to remove this.